src/desktop/apps/auth2/helpers.ts

```diff
@@ -35,6 +35,7 @@ export const handleSubmit = (
     _csrf: sd.CSRF_TOKEN,
     signupIntent: intent,
     signupReferer,
+    agreed_to_receive_emails: values.accepted_terms_of_service,
   })
```

Comment:
Possible to update this test to use current fields?
https://github.com/xtina-starr/force/blob/c8735eaf18e9c2c0490d9ced99797a3b4fbd8016/src/desktop/apps/auth2/__tests__/helpers.jest.js